src/pdf_parser/pdf_utils/disambiguator/unexplained.py

```diff
@@ -0,0 +1,65 @@
+import logging
```

Comment:
Not a major thing, but why are we putting this into it's own module. Seems a bit fuzzy as to why this isn't a util?

Reply:
also, what is this? would be useful to have a file-level docstring explaining why there's a submodule called 'unexplained'

Reply:
Moved to the utils module